src/popup/modules/UserInterface.js

```diff
@@ -167,12 +169,9 @@ function selectAllText(event) {
  * @returns {void}
  */
 function scrollToTop(event) {
-    console.info("scrollToTop", event);
-
-    if (event.target.scrollTop !== 0) {
-        event.target.scrollTop = 0;
-    }
-
+    console.info("scrollToTop", event); 
+    event.target.scrollTo(0,0);
+  
     // only retry once, if needed
     if (event.setScrolled) {
```

Comment:
Mhh, it seems I've introduced this, because before, here was a loop or so, that passed on the `event` parameter. I just guess, now it does not do that anymore. So as far as I see, it may indeed be useless, because it is first sentence, then the whole thing is discarded and the new event is a different one, when we get again here…